include/real/real_data.hpp

```diff
@@ -337,6 +338,99 @@ namespace boost {
                     break;
                 }
 
+                case OPERATION::EXPONENT :{
+                    this->_approximation_interval.lower_bound = 
+                        exponent(ro.get_lhs_itr().get_interval().lower_bound.up_to(_precision, false), _precision, false);
+                    this->_approximation_interval.upper_bound = 
+                        exponent(ro.get_lhs_itr().get_interval().upper_bound.up_to(_precision, true), _precision, true);
+                    break;
+                }
+
+                case OPERATION::LOGARITHM :{
```

Comment:
What happens when your input interval includes the zero?

Reply:
Ok. I got your point.
I will modify the function to increase the precision of input until it does not have 0 in its intervals, otherwise through out of domain error. We will replace the error with "NaN" when we have the implementation of it.